float32 laser_depth
float32 point_x
float32 point_y
float32 order
string obj_one
string obj_two
float32 isopen
